src/System.Management.Automation/resources/EventResource.resx

```diff
@@ -81,6 +81,16 @@
  InnerException: {3} 
 </value>
 </data>
+<data name="PS_PROVIDEReventE_O_ExperimentalFeatureInvalidName" xml:space="preserve">
+    <value>Experimental Feature Initialization: Ignore the experimental feature '{0}' from the config file. {1}</value>
```

Comment:
seems on that the message doesn't say the name is invalid.

Reply:
The `{1}` part will say why this name is invalid.